feature/gribi/ate_tests/base_leader_election_test/base_leader_election_test.go

```diff
@@ -288,4 +287,7 @@ func TestElectionIDChange(t *testing.T) {
 	}
 
 	testIPv4LeaderActiveChange(ctx, t, args)
+
+	// Flush all entries after test. ClientA or ClientB doesn't matter since we use Election Override on FlushAll.
+	clientA.FlushAll(t)
```

Comment:
Shouldn't most uses of FlushAll be moved to the top (after clientA.Close()) as defers so they execute if the test fails too?

Reply:
Done!